actioncable/lib/rails/generators/channel/channel_generator.rb

```diff
@@ -33,7 +33,7 @@ def file_name
           @_file_name ||= super.sub(/_channel\z/i, "")
         end
 
-        # FIXME: Change these files to symlinks once RubyGems 2.5.0 is required.
+        # FIXME: Change these files to symlinks once RubyGems 2.5.1 is required.
```

Comment:
This isn’t related to the version of Ruby required. Please remove this change.

Reply:
Sure. thanks. For now I will close the PR as discussed on the related issue.